tests/js/snapshots.test.js

```diff
@@ -1,5 +1,11 @@
 import { expect, test } from "@playwright/test";
 
+test("left side bar only uses purple for current page", async ({ page }) => {
```

Comment:
I think this test could be used to validate more than just the purple-ness, I'd recommend using a more generic test name, maybe this:

```suggestion
test("left sidebar renders correctly", async ({ page }) => {
```